spyder_kernels/comms/commbase.py

```diff
@@ -385,13 +386,24 @@ def _register_call(self, call_dict, callback=None):
         if blocking or callback is not None:
             self._reply_waitlist[call_id] = blocking, callback
 
+    def on_outgoing_call(self, call_dict):
+        """A message is about to be sent"""
+        call_dict["Pickle Highest Protocol"] = pickle.HIGHEST_PROTOCOL
```

Comment:
Please change this key to `pickle_highest_protocol` (it's a bit strange to have a key with spaces and capital letters).